src/test/java/dev/cdevents/CDEventsTest.java

```diff
@@ -22,7 +22,7 @@ void createPipelineRunFinishedEventAsCloudEvent() {
         cdEvent.setSource(URI.create("http://dev.cdevents"));
 
         cdEvent.setSubjectId("/dev/pipeline/run/subject");
-        cdEvent.setSubjectSource(URI.create("/dev/pipeline/run/subject"));
+        cdEvent.setSubjectSource(URI.create("http://dev/pipeline/run/subject"));
```

Comment:
NIT: why `http://`? It's fine, but even the string starting with `/dev` should be a valid URI.